Gitweb:     http://git.kernel.org/git/?p=linux/kernel/git/torvalds/linux-2.6.git;a=commit;h=03c086a747d0b242878eb881971ec61c1555869d
Commit:     03c086a747d0b242878eb881971ec61c1555869d
Parent:     245073f0b3a8b8b626f3222dacb483695bf46419
Author:     Darren Salt <linux@youmustbejoking.demon.co.uk>
AuthorDate: Thu Mar 13 15:35:49 2008 +0000
Committer:  Linus Torvalds <torvalds@linux-foundation.org>
CommitDate: Sat Mar 22 17:00:57 2008 -0700

    PNP: increase the number of PnP memory resources from 12 to 24
    
    Increase the number of PnP memory resources from 12 to 24.
    
    This removes an "exceeded the max num of mem resources" warning on boot. I
    also noticed the reservation of two more iomem ranges on the computer on
    which this was tested.
    
    Signed-off-by: Darren Salt <linux@youmustbejoking.demon.co.uk>
    Signed-off-by: Ingo Molnar <mingo@elte.hu>
    Acked-by: Bjorn Helgaas <bjorn.helgaas@hp.com>
    Signed-off-by: Linus Torvalds <torvalds@linux-foundation.org>
---
 include/linux/pnp.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/linux/pnp.h b/include/linux/pnp.h
index cd6332b..29dd558 100644
--- a/include/linux/pnp.h
+++ b/include/linux/pnp.h
@@ -14,7 +14,7 @@
 #include <linux/mod_devicetable.h>
 
 #define PNP_MAX_PORT		40
-#define PNP_MAX_MEM		12
+#define PNP_MAX_MEM		24
 #define PNP_MAX_IRQ		2
 #define PNP_MAX_DMA		2
 #define PNP_NAME_LEN		50

